### ISSUE TITLE
[Bug report]  The czg ai command does not work.

### ISSUE THREAD

**Issue by guotingchao** [Zhengqbbb/cz-git#123]
### ✅ Checklist

- [X] I am using the **latest version**
- [X] I have searched [FAQ](https://cz-git.qbb.sh/faq/)

### 📄 System Info

```Shell
System:
    OS: macOS 13.3
    CPU: (12) arm64 Apple M2 Max
    Memory: 956.44 MB / 32.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 19.8.1 - /opt/homebrew/bin/node
    Yarn: 3.5.0 - /opt/homebrew/bin/yarn
    npm: 9.5.1 - /opt/homebrew/bin/npm
  npmGlobalPackages:
    commitizen: 4.3.0
```

#### Concise Description
Using czg CLi in my git repository folder path, the czg ai command does not work.

#### Expected behavior

**I'm pretty sure I still have a balance of about $17 in my account, and I haven't made any calls recently, and api key is rebound.**

#### Steps to reproduce
`$> czg ai`

#### Error Message
```
? Select the type of change that you're committing: feat:     A new feature
ℹ Generating your AI commit subject...
[err]>>>: Fetch OpenAI API message failureThe response HTTP Code: 429
You exceeded your current quota, please check your plan and billing details.
```

**Comment by Zhengqbbb** [comment]
> You exceeded your current quota, please check your plan and billing details.

The error is openAI response.
I'm also pretty sure it can work ~ Please check your API key billing details:

https://platform.openai.com/account/usage

**Comment by AugusDogus** [comment]
I just experienced this issue after setting up a fresh API key with 0 usage.

![image](https://github.com/user-attachments/assets/fc31f878-cb5e-4f5b-be71-b7087634809a)

![image](https://github.com/user-attachments/assets/8aa31b51-790b-4372-a006-f66a7d0dd8c0)

**Comment by Zhengqbbb** [comment]
> I just experienced this issue after setting up a fresh API key with 0 usage.
> 

The error message is from OpenAI. please try a mini example by `curl` command: https://platform.openai.com/docs/quickstart/step-3-sending-your-first-api-request

<img width="1315" alt="image" src="https://github.com/user-attachments/assets/c1ac4a7e-67ff-4254-98b4-87c2672b5ebb">
> Just tested and everything works fine.

**Comment by AugusDogus** [comment]
Thanks for taking a look @Zhengqbbb; after further investigation this was because I did not have a paid plan and misunderstood the usage limits for a free account.

> TL;DR: You need to upgrade to a paid plan. Set up a paid account, add a credit or debit card, and generate a new API key if your old one was generated before the upgrade. It might take 10 minutes or so after you upgrade to a paid plan before the paid account becomes active and the error disappears.

Relevant [stackoverflow](https://stackoverflow.com/a/75898717).

Might be worth noting this in the readme - I'm happy to open a PR if you think this would be useful.
